pkg/70ae8859-888c-47a4-9d26-d2486c6c3a14/package.lp

```diff
@@ -0,0 +1,180 @@
+   (vertex (position -2.43 3.0) (angle 0.0))
+   (vertex (position 2.43 3.0) (angle 0.0))
+   (vertex (position 2.43 -3.0) (angle 0.0))
+   (vertex (position -2.43 -3.0) (angle 0.0))
+   (vertex (position -2.43 -1.175) (angle 0.0))
+  )
+  (polygon 3dfe7e17-dc6d-4917-aa4f-18ab1a91f196 (layer top_documentation)
+   (width 0.0) (fill true) (grab_area false)
+   (vertex (position -0.44 -2.34) (angle 0.0))
+   (vertex (position -0.44 -2.74) (angle 0.0))
+   (vertex (position -2.75 -2.74) (angle 0.0))
+   (vertex (position -2.75 -2.34) (angle 0.0))
+   (vertex (position -0.44 -2.34) (angle 0.0))
+  )
+  (polygon 260a99fd-dbcd-4614-b631-4c0e435ba1d4 (layer top_documentation)
+   (width 0.0) (fill true) (grab_area false)
+   (vertex (position 0.44 -1.07) (angle 0.0))
+   (vertex (position 0.44 -1.47) (angle 0.0))
+   (vertex (position 2.75 -1.47) (angle 0.0))
+   (vertex (position 2.75 -1.07) (angle 0.0))
+   (vertex (position 0.44 -1.07) (angle 0.0))
+  )
+  (polygon 2e97628b-dfd8-4f82-9b7f-d6447eb97617 (layer top_placement)
+   (width 0.25) (fill false) (grab_area false)
+   (vertex (position -3.03 3.195) (angle 0.0))
+   (vertex (position 2.68 3.195) (angle 0.0))
+  )
+  (polygon 4a2b3b15-9b92-44f6-8c8b-643d48a36258 (layer top_documentation)
+   (width 0.0) (fill true) (grab_area false)
+   (vertex (position 0.44 2.74) (angle 0.0))
+   (vertex (position 0.44 2.34) (angle 0.0))
+   (vertex (position 2.75 2.34) (angle 0.0))
+   (vertex (position 2.75 2.74) (angle 0.0))
+   (vertex (position 0.44 2.74) (angle 0.0))
+  )
+  (polygon 53830e59-8224-40cc-8d3d-a53a9aeec5c8 (layer top_documentation)
+   (width 0.0) (fill true) (grab_area false)
+   (vertex (position -0.44 1.47) (angle 0.0))
+   (vertex (position -0.44 1.07) (angle 0.0))
+   (vertex (position -2.75 1.07) (angle 0.0))
+   (vertex (position -2.75 1.47) (angle 0.0))
+   (vertex (position -0.44 1.47) (angle 0.0))
+  )
+  (polygon 9b32c7ae-4b22-4b3a-98a1-21d5de45b3c1 (layer top_courtyard)
+   (width 0.25) (fill false) (grab_area false)
+   (vertex (position -3.405 3.57) (angle 0.0))
+   (vertex (position 3.405 3.57) (angle 0.0))
+   (vertex (position 3.405 -3.57) (angle 0.0))
+   (vertex (position -3.405 -3.57) (angle 0.0))
+   (vertex (position -3.405 3.57) (angle 0.0))
+  )
+  (polygon 66d42cd3-8bda-49e6-a570-e806dd93d474 (layer top_documentation)
+   (width 0.0) (fill true) (grab_area false)
+   (vertex (position 0.44 1.47) (angle 0.0))
+   (vertex (position 0.44 1.07) (angle 0.0))
+   (vertex (position 2.75 1.07) (angle 0.0))
+   (vertex (position 2.75 1.47) (angle 0.0))
+   (vertex (position 0.44 1.47) (angle 0.0))
+  )
+  (polygon b2616bf5-03d0-4ceb-b93d-d751a121b3ea (layer top_hidden_grab_areas)
+   (width 0.0) (fill true) (grab_area true)
+   (vertex (position -2.555 3.195) (angle 0.0))
+   (vertex (position 2.555 3.195) (angle 0.0))
+   (vertex (position 2.555 -3.195) (angle 0.0))
+   (vertex (position -2.555 -3.195) (angle 0.0))
+   (vertex (position -2.555 3.195) (angle 0.0))
+  )
```

Comment:
Hmm actually we almost never add grab areas to footprints because currently they make it very hard to select objects under the footprint (e.g. a trace on the other PCB side). As long as this issue exists I'd prefer to remove the grab area.

```suggestion
```